tests/OrganizationsMetadataTest.php

```diff
@@ -154,7 +156,10 @@ public function testMaintainerAdminCanManageUserMetadata(): void
 
         $this->client->removeUserFromMaintainer($this->testMaintainerId, $this->normalUser['id']);
 
-        $this->cannotManageUserMetadata($this->normalUserClient);
+        $this->cannotManageUserMetadata($this->normalUserClient, $metadataArray[0]['id']);
```

Comment:
Tu mozes najskor, ze to ide zmazat (napr si zmazes jedno metadato a skontorlujes ze tam druhe ostalo) + potom testujes to co tam vzdy mas ze ked mu odoberies tu rolu tak ze to nejde

Reply:
vyreseno prehozenim a rozdelenim skrze komentare